customs.js

```diff
@@ -55,6 +57,13 @@
       }
     )
     .then(
+      (function (result) {
+        if (result.lockout) {
+          return this.db.lockAccount(account)
+        }
+      }).bind(this)
+    )
+    .then(
       function () {},
       function (err) {
         log.error({ op: 'customs.flag.1', email: email, err: err })
```

Comment:
@chilts yeah, I know what you mean, it definitely smells like some sort of layering/abstraction violation.

FWIW the current structure is my fault (from a previous PR on this topic) and my reasoning at the time was: it's better to have the lockout handled in one place, rather than have to remember to do it each time you call `customs.flag()`.

For example, you said "the higher level `routes/account.js` should make that decision", but there is also a call to `customs.flag()` in `routes/password.js` where we would have to duplicate any such logic.

I wonder if there's a better abstraction that would satisfy from both perspectives?